test/mindwendel_web/live/label_live/captions_test.exs

```diff
@@ -1,11 +1,13 @@
 defmodule MindwendelWeb.LabelLive.CaptionsTest do
-  alias MindwendelWeb.LabelLive.CaptionsComponent
-  alias Mindwendel.Brainstormings
-  use MindwendelWeb.ConnCase
+  use MindwendelWeb.ConnCase, async: true
+
   import Phoenix.LiveViewTest
```

Comment:
There is a lint for this (which we could activate), but normally modules follow a module layout something along those lines:

1. moduledoc
2. use
3. import
4. require
5. alias

That is, sort of in order of importance/what it means to the module (esp. the last 4, moduledoc is more convenience) - as `use` does more than `import` does more than `require` does more than `alias` as for what the module does. Keeping them in that order helps with reading the module imo :)

Reply:
I think it's a good idea to activate such a lint. But I would suggest doing it in another PR :)